cmd/otel-allocator/watcher/main.go

```diff
@@ -2,16 +2,17 @@ package watcher
 
 import (
 	"fmt"
+	"github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/allocation"
```

Comment:
I think this import is in the wrong place.  This is the first file I noticed it, but there's a similar issue in other files as well.